src/features/onboarding/accountCreatePassphrase/AccountCreatePassphraseScreen.tsx

```diff
@@ -36,7 +36,7 @@ const AccountCreatePassphraseScreen = () => {
 
   useEffect(() => {
     const viewedAll = viewedWords.every((w) => w)
-    if (!viewedAll) return
+    if (!viewedAll && !__DEV__) return
```

Comment:
So this is just to skip the check during dev?

Reply:
correct.